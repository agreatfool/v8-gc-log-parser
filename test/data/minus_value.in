[52603:0x102800000] [I:0x102800000]       45 ms: pause=0.7 mutator=-1501275000064.5 gc=s external=0.0 mark=0.0 sweep=0.00 sweepns=0.00 sweepos=0.00 sweepcode=0.00 sweepcell=0.00 sweepmap=0.00 evacuate=0.0 new_new=0.0 root_new=0.0 old_new=0.0 compaction_ptrs=0.0 intracompaction_ptrs=0.0 misc_compaction=0.0 weak_closure=0.0 inc_weak_closure=0.0 weakcollection_process=0.0 weakcollection_clear=0.0 weakcollection_abort=0.0 total_size_before=2277616 total_size_after=2006608 holes_size_before=0 holes_size_after=0 allocated=2277616 promoted=0 semi_space_copied=749256 nodes_died_in_new=0 nodes_copied_in_new=4 nodes_promoted=0 promotion_ratio=0.0% average_survival_ratio=73.4% promotion_rate=0.0% semi_space_copy_rate=73.4% new_space_allocation_throughput=0 context_disposal_rate=0.0 steps_count=0 steps_took=0.0 scavenge_throughput=1472242 