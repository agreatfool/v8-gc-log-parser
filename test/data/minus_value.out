[
  {
    "pid": 52603,
    "isolate": "0x102800000",
    "type": "nvp",
    "time_since_init": 45,
    "data": {
      "pause": 0.7,
      "mutator": -1501275000064.5,
      "gc": "s",
      "external": 0,
      "mark": 0,
      "sweep": 0,
      "sweepns": 0,
      "sweepos": 0,
      "sweepcode": 0,
      "sweepcell": 0,
      "sweepmap": 0,
      "evacuate": 0,
      "new_new": 0,
      "root_new": 0,
      "old_new": 0,
      "compaction_ptrs": 0,
      "intracompaction_ptrs": 0,
      "misc_compaction": 0,
      "weak_closure": 0,
      "inc_weak_closure": 0,
      "weakcollection_process": 0,
      "weakcollection_clear": 0,
      "weakcollection_abort": 0,
      "total_size_before": 2277616,
      "total_size_after": 2006608,
      "holes_size_before": 0,
      "holes_size_after": 0,
      "allocated": 2277616,
      "promoted": 0,
      "semi_space_copied": 749256,
      "nodes_died_in_new": 0,
      "nodes_copied_in_new": 4,
      "nodes_promoted": 0,
      "promotion_ratio": 0,
      "average_survival_ratio": 73.4,
      "promotion_rate": 0,
      "semi_space_copy_rate": 73.4,
      "new_space_allocation_throughput": 0,
      "context_disposal_rate": 0,
      "steps_count": 0,
      "steps_took": 0,
      "scavenge_throughput": 1472242
    }
  }
]
